dbt/models/iasworld/schema/iasworld.owndat.yml

```diff
@@ -221,30 +221,8 @@ sources:
             description: '{{ doc("column_whocalc") }}'
           - name: zip1
             description: '{{ doc("shared_column_mail_address_zipcode_1") }}'
-            tests:
-              - not_accepted_values:
-                  name: iasworld_owndat_zip1_has_no_00000_values
```

Comment:
My understanding is that these tests are no longer necessary, since address tests should be defined on `legdat` (which they already are).